demisto_sdk/commands/pre_commit/hooks/docker.py

```diff
@@ -385,7 +397,14 @@ def generate_hooks(
         ) in object_to_files_with_objects.items():
             change_working_directory = False
             files = {file for file, _ in files_with_objects}
+            objects_ = [object_ for _, object_ in files_with_objects]
             hook = deepcopy(new_hook)
+            if new_hook["name"].startswith("mypy-in-docker"):
```

Comment:
consider with @YuvHayun whether we can add some `Hook`-like mechanism to modify docker hooks like we do to non-docker hooks. Otherwise we complicate the code here with patches.